frameworks/V/pico.v/pico.v.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:sid
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install wget unzip build-essential ca-certificates --no-install-recommends --yes
```

Comment:
can you add -yqq to update and install to quiet them a bit

Reply:
done